src/components/Renderer/index.tsx

```diff
@@ -153,7 +184,8 @@ const RendererBase = ({
 		uiSchema,
 		extraContext,
 	});
-	const processedValue = getValue(value, schema, processedUiSchema);
+
+	const processedValue = getValue(value, schema, uiSchema);
```

Comment:
Is it equivalent that we are now passing `uiSchema` instead of `processedUiSchema`?